plugins/modules/cloud/google/gcp_storage_aws_transfer_job.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Terry Jones <terry.jones@example.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: gcp_storage_aws_transfer_job
+version_added: 2.10
```

Comment:
```suggestion
version_added: 1.0.0
```
The next release of this collection will be 1.0.0.